libcrux-ml-kem/src/polynomial.rs

```diff
@@ -14,7 +14,7 @@ pub(crate) const ZETAS_TIMES_MONTGOMERY_R: [i16; 128] = [
 pub(crate) const VECTORS_IN_RING_ELEMENT: usize =
     super::constants::COEFFICIENTS_IN_RING_ELEMENT / FIELD_ELEMENTS_IN_VECTOR;
 
-#[cfg_attr(eurydice, derive(Clone, Copy))]
+#[derive(Clone, Copy)]
 pub(crate) struct PolynomialRingElement<Vector: Operations> {
```

Comment:
This is a little unfotunate. How about keeping the `eurydice` guard and adding a second one with `#[cfg_attr(not(eurydice), derive(Clone))]` and manually clone in the unpacked key generation? That way it's explicit at least.